test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -30,15 +30,15 @@ def test_ec2_tensorflow_inference_cpu(tensorflow_inference, ec2_connection, regi
 
 
 @pytest.mark.integration("elastic_inference")
-@pytest.mark.model("half_plus_two")
+@pytest.mark.model("mnist")
```

Comment:
Are we labeling it incorrectly?

Reply:
Yes, it was previously named incorrectly.